kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/KafkaConnectService.java

```diff
@@ -265,8 +265,9 @@ private Mono<Void> restartTasks(KafkaCluster cluster, String connectName,
                                   String connectorName, Predicate<TaskDTO> taskFilter) {
     return getConnectorTasks(cluster, connectName, connectorName)
         .filter(taskFilter)
-        .flatMap(t ->
-            restartConnectorTask(cluster, connectName, connectorName, t.getId().getTask()))
+        .flatMap(t -> restartConnectorTask(cluster, connectName, connectorName, t.getId().getTask())
+            .onErrorResume(e -> Mono.error(new RuntimeException("Failed to restart task", e)))
```

Comment:
The issue suggests that we want to display this information on frontend, which requires to return an exception via API